rero_invenio_base/cli/es/index.py

```
@@ -235,9 +265,9 @@ def update_mapping(aliases, settings):
                         if res.get("acknowledged"):
                             click.secho(f"index: {index} has been successfully updated", fg="green")
                         else:
-                            click.secho(f"error: {res}", fg="red")
-                except Exception as excep:
-                    click.secho(f"error: {excep}", fg="red")
+                            click.secho(f"error: {index}: {res}", fg="red")
+                except Exception as err:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)